src/viam/sdk/common/utils.cpp

```diff
@@ -76,6 +76,18 @@ google::protobuf::Timestamp time_pt_to_timestamp(
     return timestamp;
 }
 
+google::protobuf::RepeatedField<double> vector_to_repeated_double_field(
+    const std::vector<double>& v) {
+    google::protobuf::RepeatedField<double> rf = {v.begin(), v.end()};
+    return rf;
```

Comment:
Also, I'll note that since this is now in `util`, you could make this more general by making it a template:

```
template<typename T>
google::protobuf::RepeatedField<T> vector_to_repeated_field(const std::vector<T>& v) { return {v.begin(), v.end()}; }
```

Which honestly has me wondering a bit about whether the utility function is really bringing much to the table.

Reply:
Yeah I thought this was a similar issue to the concatenation of vectors requiring more verbosity (see earlier thread about `get_geometries` for `MyBase`). Looks like your suggestion works just fine. I've removed both of these utility functions given that they add more lines that just calling the one-liners you've suggested.